### ISSUE TITLE
Forgot password translations

### ISSUE THREAD

**Issue by gicat** [meteor-useraccounts/core#343]
Hi,

I am having issues with translations when using 'Forgot Password' functionality. The email sent to the user is in english and the dialogs for setting new password and the following dialog that confirms the correct operation are in english. The client behaves correctly with the language I have chosen for my site (italian).
What can I do to translate these messages and customize email content sent to the client in case of forgot password?

Regards,
Giuseppe

**Comment by TheRedDevil** [comment]
Try putting this somewhere on server, it should do the work. Both methods have user as param, and the method for body part also has url which represents password reset link.

```javascript
Accounts.emailTemplates.resetPassword.subject = function (user) {
    return "SUBJECT";
};

Accounts.emailTemplates.resetPassword.text = function (user, url) {
    return "Reset password by clicking " + url;
};
```

Regarding form labels, I found out how to change message that appears when user successfully sets password, so I guess you can change other messages in similar manner:

```javascript
AccountsTemplates.configure({
texts: {
        info: {
            pwdSet: T9n.get('passwordSet')
        }
    }
});
```

**Comment by gicat** [comment]
Thank you. 

For what concerns email templates text it works as you suggested. Starting from your suggestion I have found this resource that explains it: https://gentlenode.com/journal/meteor-20-verify-an-email-with-meteor-accounts/42

For labels it doesn't work for forgot password. I am going to further investigate to better understand the issue.

Regards,
Giuseppe

**Comment by splendido** [comment]
Ciao Giuseppe,

some [accounts-t9n](https://github.com/softwarerero/meteor-accounts-t9n) keys are still to be translated and are currently missing.

Please see #123 and [Define Translations](https://github.com/softwarerero/meteor-accounts-t9n#define-translations) for accounts-t9n.

**Comment by splendido** [comment]
@gicat let me know if you need further help!

**Comment by gicat** [comment]
Ciao Luca,

the issue seems not related to meteor-useraccounts, but to meteor/packages/accounts-ui-unstyled/login_buttons_dialogs.html. Here text are not managed by any framework, but in plain english. For example the confirmation dialog (<template name="_resetPasswordDialog">) that is opened after clicking on forgot password link received in email, is in english.
I will ask to Meteor team how to manage this problem.

Thanks.
Giuseppe

**Comment by splendido** [comment]
oh, ok.
I though you were using `useraccounts`.

Ciao!

**Comment by gicat** [comment]
Yes, I am using useraccounts, I have configured AccountTemplates to display the showForgotPasswordLink. For what I understood, the form for managing the forgot password use case is from useraccount, while the part that manages the dialog for changing the password from the email link that arrives to the user, is contained in accounts-ui project of Meteor. I haven't found any code that contains this logic inside useraccounts. Is this correct?

Ciao,
Giuseppe

**Comment by splendido** [comment]
...oh, in this case: it's all about useraccounts!

make sure you get a link like this:
`http://localhost:3000/reset-password/jEoy9dwo48kckijpH9wbU3g_TluR-XWJoxvp1h6Url7`

...and remove `accounts-ui` ;-)

**Comment by gicat** [comment]
Thank you, now it works correctly. I hadn't removed accounts-ui.
Just another question: in the generated url that arrives in the email the url has the following format: 
http://localhost:3000/#/reset-password/jEoy9dwo48kckijpH9wbU3g_TluR-XWJoxvp1h6Url7
I don't know where '#' came from. Did you experienced such issue? For the moment i will parse the url and remove the '#'.

Thanks a lot.
Ciao,
Giuseppe

**Comment by splendido** [comment]
make sure you [configured all necessary routes](https://github.com/meteor-useraccounts/boilerplates/blob/master/bootstrap/lib/config/at_config.js#L2-8)

**Comment by AhmyOhlin** [comment]
i have the same issue  with hash '#' in the link 
how can i fix this problem
